docs/how-to/build-a-12-factor-app-rock.rst

```diff
@@ -0,0 +1,171 @@
+How to build a 12-Factor app rock
+*********************************
+
+Include extra files in the OCI image
+------------------------------------
+
+The following files are included in the image by default from
+the root of the project:
+
+- ``app`` (does not apply to the ``go-framework``)
+- ``app.py`` (does not apply to the ``go-framework``)
+- ``migrate``
+- ``migrate.sh``
+- ``migrate.py`` (does not apply to the ``go-framework``)
+- ``static``
+- ``templates``
+
+To change this, the following snippet needs to be added to
+the ``rockcraft.yaml``:
+
+.. tabs::
+
+   .. group-tab:: Flask
+
+      .. code-block:: yaml
+
+           parts:
+             flask-framework/install-app:
+               prime:
+                 - flask/app/.env
+                 - flask/app/app.py
+                 - flask/app/webapp
+                 - flask/app/templates
+                 - flask/app/static
+
+      Note the ``flask/app/`` prefix that is required followed by the relative path to
+      the project root.
+
+   .. group-tab:: Django
+
+      .. code-block:: yaml
```

Comment:
@jdkandersson @javierdelapuente I added this in for Django as I suspected that the empty tab was causing rendering issues (not clear if that's really the case, but I digress). Please take a look and verify that this is correct for Django -- otherwise I can edit or remove. Let me know, thanks

Reply:
I think this is N/A for now because we copy the whole directory, i.e., we don't need to specify any specific files

Reply:
Thanks for checking; updated